python/composio/tools/toolset.py

```diff
@@ -145,9 +145,11 @@ class ComposioToolSet(WithLogger):  # pylint: disable=too-many-public-methods
 
     def __init_subclass__(
         cls,
+        *args: t.Any,
         runtime: t.Optional[str] = None,
         description_char_limit: t.Optional[int] = None,
         action_name_char_limit: t.Optional[int] = None,
```

Comment:
Consider adding docstring to document the new `action_name_char_limit` parameter, its purpose, and expected values. This would help other developers understand the parameter's role and constraints.